python/pyspark/ml/tests.py

```diff
@@ -772,6 +798,25 @@ def test_nested_pipeline_persistence(self):
             except OSError:
                 pass
 
+    def test_onevsrest(self):
```

Comment:
This is similar to the persistence test in OneVsRestTests.  Can you please remove the old one?